src/app/public/src/modules/code-block/code-block.component.scss

```diff
@@ -13,7 +13,31 @@
   margin-bottom: $stache-margin * 2;
 }
 
-/deep/
+.stache-code-block {
+  position: relative;
+
+  ::ng-deep
+  .stache-clipboard-btn {
```

Comment:
More of a question: should we put these styles in the clipboard style sheet to maintain consistency for when it's used outside of the code block?

Reply:
These styles are very particular to the code-block itself.  The styles of the button itself are just those of the sky button.  I think consuming components should be able to position the button however it is ideal for that component.  
e.g  a form input wouldn't want to have the button in the same place as the code block does.